operator/Makefile

```
@@ -38,7 +38,7 @@ help: Makefile
 # Requires operator-sdk - download it from https://github.com/operator-framework/operator-sdk/releases
 operator-build: .ensure-operator-sdk-exists
 	@echo Build operator
-	${OP_SDK} build ${OPERATOR_IMAGE_NAME}:${OPERATOR_IMAGE_VERSION}
+	${OP_SDK} build "${OPERATOR_IMAGE_NAME}":${OPERATOR_IMAGE_VERSION}
```

why wouldn't you wrap the whole thing in quotes?
```suggestion
	${OP_SDK} build "${OPERATOR_IMAGE_NAME}:${OPERATOR_IMAGE_VERSION}"
```

---

I did what I needed to do to get it working but enclosing the whole thing should be fine, I'll change it.